src/java/org/apache/cassandra/streaming/ProgressInfo.java

```diff
@@ -110,7 +110,7 @@ public String toString(boolean withPorts)
     {
         StringBuilder sb = new StringBuilder(fileName);
         sb.append(" ").append(currentBytes);
-        sb.append("/").append(totalBytes).append(" bytes");
+        sb.append("/").append(totalBytes).append(" bytes ");
```

Comment:
looks like this

```
netstats_test/test_table 528363/528363 bytes (100%) received from idx:0/127.0.0.1
```

instead of 

```
netstats_test/test_table 528363/528363 bytes(100%) received from idx:0/127.0.0.1
```